tagged-manifest.xml

```diff
@@ -19,13 +19,13 @@
   <project name="android_frameworks_base" path="frameworks/base" remote="sony-patches" revision="de9587cbab4f17c7744e7ef721c54aad9680d0f6" upstream="sony-aosp-6.0.1_r80-20170902"/>
   <project name="android_frameworks_native" path="frameworks/native" remote="hybris-patches" revision="eb62b481f713c3a3c5d32a2ece78fa45b9537e46" upstream="hybris-sony-aosp-6.0.1_r80-20170902"/>
   <project name="android_hardware_broadcom_libbt" path="hardware/broadcom/libbt" remote="sony-patches" revision="97490d1923dc459a7502c9f488d55a8c4577caf7" upstream="sony-aosp-6.0.1_r80-20170902"/>
-  <project name="android_hardware_qcom_audio" path="hardware/qcom/audio" remote="sony-patches" revision="d287a046899d47a60c9728946982f65e0b85ceb4" upstream="sony-aosp-6.0.1_r80-20170902"/>
+  <project name="android_hardware_qcom_audio" path="hardware/qcom/audio" remote="sony-patches" revision="24be7b102d897f02824d6a326adc7c52ae21b42e" upstream="sony-aosp-6.0.1_r80-20170902"/>
```

Comment:
This one is not update from upstream, but https://github.com/mer-hybris/android_hardware_qcom_audio/pull/1, so should make it to the changelog